pkg/encryption/utils_test.go

```diff
@@ -130,12 +130,12 @@ func TestValidate(t *testing.T) {
 	assert.Equal(t, validValue, expectedValue)
 }
 
-func TestGenerateRandomASCIIString(t *testing.T) {
-	randomString, err := GenerateRandomASCIIString(96)
+func TestGenerateCodeVerifierString(t *testing.T) {
+	randomString, err := GenerateCodeVerifierString(96)
 	assert.NoError(t, err)
 
-	// Only 8-bit characters
-	assert.Equal(t, 96, len([]byte(randomString)))
+	// Should be 128 characters long
+	assert.Equal(t, 128, len([]byte(randomString)))
```

Comment:
Could this be a breaking change for any providers 🤔 They should all follow the RFC but we have seen the weirdest things in the past. On the other hand I don't want to introduce another flag to make this configurable if it might not be neccesary.

Therefore we should just mention it in the next release letter